test/expected/timestamp.out

```diff
@@ -2006,10 +2006,16 @@ SELECT test.interval_to_internal('BIGINT'::regtype, 2147483649::bigint);
            2147483649
 (1 row)
 
+-- Default interval for integer is supported as part of
```

Comment:
Could you also test the two other default values?

Reply:
Added.